task/06-conditions-n-loops-tasks.js

```diff
@@ -296,10 +354,37 @@ function getDigitalRoot(num) {
  *   '{[(<{[]}>)]}' = true 
  */
 function isBracketsBalanced(str) {
-    throw new Error('Not implemented');
-}
 
 
+  let brackets = ['[', '{', '(', '<', ']', '}', ')', '>'];
+
+  str = str.split('');
+
+  while(str.length) {
+    let comapreStr = str;
+    let i = 1;
+
+    while (i < str.length) {
```

Comment:
It still pretty much the same code. And you're still iteraterting over the str which you change inside the loop